### PR TITLE
Add jest

### PR DESCRIPTION
**Pull request by paradime**

### CODE REVIEW

File: src/routes/Group/components/GroupView.js [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import "./GroupView.scss";
+//import "./GroupView.scss";
```
Comment (jarrydlee) [inline src/routes/Group/components/GroupView.js]: Is this meant to still be commented out?

File: src/store/group.js [hunk 2]
```diff
@@ -14,6 +14,7 @@ export const fetchGroup = (groupId) => {
   return (dispatch, getState) => {
     return new Promise((resolve) => {
       getJSON('group/' + groupId).then((response) => {
+        console.log(JSON.stringify(response))
```
Comment (jarrydlee) [inline src/store/group.js]: Assuming we don't need this anymore

File: __tests__/group_spec.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
...mail.com","type":"email","name":"at","__v":0,"createdAt":"2017-02-12T17:00:07.368Z"}],"groups":["AudraDickinson"],"auth":null},{"_id":"58a09497b73f14c7d6d64438","name":"Demetrius.Hahn","email":"Jean.Lemke52@hotmail.com","role":0,"__v":3,"createdAt":"2017-02-12T17:00:07.376Z","isSysAdmin":false,"delays":[1],"devices":[{"_id":"58a09498b73f14c7d6d64527","contactInformation":"4674307718","type":"phone","name":"officia","__v":0,"createdAt":"2017-02-12T17:00:07.368Z"},{"_id":"58a09497b73f14c7d6d644b0","contactInformation":"Carley.Hagenes85@hotmail.com","type":"email","name":"quo","__v":0,"createdAt":"2017-02-12T17:00:07.368Z"}],"groups":["AudraDickinson"],"auth":null},{"_id":"58a09497b73f14c7d6d6443a","name":"Seth4","email":"Jovanny_Block@yahoo.com","role":0,"__v":3,"createdAt":"2017-02-12T17:00:07.376Z","isSysAdmin":false,"delays":[1],"devices":[{"_id":"58a09498b73f14c7d6d64528","contactInformation":"1913833319","type":"phone","name":"facere","__v":0,"createdAt":"2017-02-12T17:00:07.368Z"},{"_id":"58a09497b73f14c7d6d644b3","contactInformation":"Maxwell20@hotmail.com","type":"email","name":"magni","__v":0,"createdAt":"2017-02-12T17:00:07.368Z"}],"groups":["AudraDickinson"],"auth":null}]}
```
Comment (jarrydlee) [inline __tests__/group_spec.js]: Would it make sense to pull these into another file so they need to be used by other tests?

**Comment by kwm4385** [comment]
Huh I looked over this before but I guess I forgot to comment. Look good.

**Pull request merged**